bindings/wasm/bindings.d.ts

```diff
@@ -5,29 +5,140 @@ type SimplePolygon = Vec2[];
 type Polygons = SimplePolygon[];
 
 declare class Manifold {
-  transform(matrix: Matrix4x3): Manifold;
+  /**
+   * Transform this Manifold in space. The first three columns form a 3x3 matrix
```

Comment:
Maybe we should mention that the matrix is row major? (I was confused about this previously)